### DISCUSSION TITLE
"Open with" for multiple files

### DISCUSSION THREAD

**Discussion by ghost** [doublecmd/doublecmd#1430] [Suggestions]
On the Mac, it would be great if the user could select several files of the same type, e.g. several JPG files, and then perform right-click > "Open with", so that all the JPG files will be opened in the selected application.

Finder supports this too.

**Comment by rich2014** [comment]
@Andreas0602 

supported in the snapshots.

**Comment by ghost** [comment]
It works great! Thank you very much, @rich2014!
